test/tests/materials/functor_properties/bc/bc.i

```diff
@@ -33,9 +33,10 @@
 
 [Materials]
   [functor]
-    type = VarFunctorMaterial
-    mat_prop_name = 'prop'
-    var = 'v'
+    type = ADGenericFunctorMaterial
+    prop_names = 'prop'
+    prop_values = 'v'
+    execute_on = 'LINEAR NONLINEAR'
```

Comment:
Is this necessary now?

Reply:
just for awareness

Reply:
now it is, since we are always by default